src/scripts/OutSystems/OSUI/Utils/PreviewInDevices/OnPostMessage.ts

```diff
@@ -1,6 +1,33 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-namespace OutSystems.OSUI.Utils.LayoutPrivate {
+namespace OutSystems.OSUI.Utils.PreviewInDevices {
+	/**
+	 * Responsible for handling the Preview In Device behavior.
+	 *
+	 * @abstract
+	 * @class OnPostMessage
+	 */
 	abstract class OnPostMessage {
+		/**
+		 * Holds the knowledge if the app is running within the Preview In Devices.
+		 *
+		 * @private
+		 * @static
+		 * @memberof OnPostMessage
+		 */
+		private static _isInPreviewInDevices = false;
+
+		/**
+		 * Adds a css class to the body, to make it easily style accordingly.
+		 *
+		 * @private
+		 * @static
+		 * @memberof OnPostMessage
+		 */
+		private static _addInPreviewCssClass(): void {
+			OnPostMessage._isInPreviewInDevices = true;
```

Comment:
Shouldn't this be at the end of the _messageFromPreview() method?

Reply:
Feels weird to set it as true, on a method that only adds a class

Reply:
Can be, there's no specific advantage in either of the ways.